packages/tools/src/utilities/stackPrefetch/stackPrefetch.ts

```diff
@@ -16,7 +16,13 @@ const addToBeginning = true;
 
 let configuration = {
   maxImagesToPrefetch: Infinity,
-  preserveExistingPool: false,
+  // Preserving the existing pool should be the default behaviour, as there might
```

Comment:
The new position aware cache deals with this a lot better by being aware of how the prefetch gets allocated/setup.  This will cause some slow down when prefetching a large display set which isn't in view any longer, but for now, I think this is a reasonable change.